content/guides/tasks/get.md

```diff
@@ -0,0 +1,16 @@
+---
+rank: 2
+related_endpoints:
+  - get_tasks_id
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Get information about a task
+
+To get information about an assigned task, call the
+[`GET /tasks/:task_id`](e://get_tasks_id) API with the
+`id` of the task.
+
+<Samples id='get_tasks_id' />
```

Comment:
Do you think it might be useful for every one of these if we also show what resource it returns? I could create an element for this, e.g. `<Resource id='task' />`

Reply:
It would reduce clicks and open pages. That seems like a win

Reply:
Ok let's do this at a later date. I'm throwing this on my backlog.